app/design/adminhtml/Magento/backend/web/css/styles-old.less

```diff
@@ -5200,3 +5200,8 @@
         page-break-after: avoid;
     }
 }
+
+.form-order-info {
+    max-width: 700px;
```

Comment:
Can you please move the styles update to Magento_Sales module styles in black and luma theme and use a variable for max-width if possible

Reply:
Hi @sivaschenko 
I have updated the changes. Can you please review?